tests/repository/metadata/meta_common.py

```diff
@@ -970,6 +970,14 @@ def test_get_instances__full(meta: MetadataRepository, created_image, created_en
     assert meta.get_instances(created_image, created_environment) == [created_instance]
 
 
+def test_get_instance__only_image(meta: MetadataRepository, created_image, created_instance):
```

Comment:
For both this and following tests here I propose to add a sibling test which checks that nothing is returned if instance was not created.

Reply:
Similar to `test_get_instances__empty` and `test_get_instances__full`.